protobuf/coreapi/api.proto

```diff
@@ -358,29 +347,39 @@ message GetServiceReply {
 }
 
 // The request's data for `ServiceLogs` API.
+//
+// **Example**
+// ```json
+// {
+//   "serviceID": "__SERVICE_ID__",
+//   "dependencies": "__SERVICE_DEPENDENCIES__"
```

Comment:
this is a `repeated string` so i'm not sure we can pass a string here, it should be an array

Reply:
Don’t forget about that one or maybe it’s normal?

Reply:
this is also done, forgot add the comment about this :)

Reply:
Are you sure you committed it ? I can't see it, I'm talking about replacing the example of `ServiceLogsRequest` with
```
{
  "serviceID": "__SERVICE_ID__",
  "dependencies": ["__DEPENDENCY_X__"]
}
```
Putting the array for the repeated string

Reply:
oh got it. do you think we should do same for the chunks(byte arrays)

Reply:
done

Reply:
actually, the data we're sending is a [byte slice](https://github.com/mesg-foundation/core/blob/dev/protobuf/coreapi/api.pb.go#L726). _bytes_ in protobufs is equal to a byte array and a _repeated bytes_  would be the equivalent of _array of byte arrays_. I think it makes sense to update them as arrays